product_secondary_unit/models/product_second_unit.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProductSecondaryUnit(models.Model):
+    _name = "product.secondary.unit"
+    _description = "Product Secondary Unit"
+
+    name = fields.Char(required=True, translate=True)
+    code = fields.Char()
+    product_tmpl_id = fields.Many2one(
+        comodel_name="product.template", string="Product Template", required=True
+    )
+    uom_id = fields.Many2one(
+        comodel_name="uom.uom",
+        string="Secondary Unit of Measure",
+        required=True,
+        help="Default Secondary Unit of Measure.",
+    )
+    factor = fields.Float(
+        string="Secondary Unit Factor", default=1.0, digits=0, required=True
```

Comment:
Can you maintain the format from last  version??

Reply:
This is done automatically by black